webapp/lib/app/configurator/standard_messages/controller_messages_helper.dart

```diff
@@ -92,11 +95,17 @@ class StandardMessagesManager {
         unsavedGroupIds.add(storageMessage.groupId);
         unsavedCategoryIds.add(storageMessage.categoryId);
       } else if (storageMessage == null) { // message added
-        editedMessages.add(localMessage);
-        unsavedMessageTextIds.add(localMessage.suggestedReplyId);
-        unsavedMessageTranslationIds.add(localMessage.suggestedReplyId);
-        unsavedGroupIds.add(localMessage.groupId);
-        unsavedCategoryIds.add(localMessage.categoryId);
+        if (localMessage.text != "" || localMessage.translation != "") {
```

Comment:
the Dart style guide encourages using `.isNotEmpty()` instead of `!=` for these kinds of checks - can you apply it to all the conditions below?